src/eureka/lib/gaussian_min.py

```diff
@@ -0,0 +1,207 @@
+    img : 2D ndarray
+        Array containing the star image.
+    meta : eureka.lib.readECF.MetaClass
+        The metadata object.
+
+    Returns 
+    ------- 
+    x : float 
+        First guess of x centroid position. 
+    y : float 
+        First guess of y centroid position.
+
+    Notes
+    -----
+    History:
+
+    - Feb 22, 2023 Isaac Edelman 
+        Initial implementation.
+    """
+    # Create median frame from Integration
+    median_Frame = np.ma.median(img, axis=0)
+
+    # Create initial centroid guess using specified method
+    if meta.centroid_tech.lower() in ['com', '1dg', '2dg']:
+        cent_func = getattr(sys.modules[__name__], 
+                            ("centroid_" + meta.centroid_tech.lower()))
+        x, y = cent_func(median_Frame)
+    else:
+        print("Invalid centroid_tech option")
+
+    # Returns x,y coordinates of centroid position
+    return x, y
+
+
+def mingauss(img, yxguess, meta):
+    """
+    Using an inital centroid guess, 
+    get a more precise centroid and PSF-width measurement 
+    using only pixels near the inital guess.
+    
+    Parameters
+    ----------
+    img : 2D ndarray
+        Array containing the star image.
+    yxguess : tuple
+        A guess at the y and x centroid positions.
+    meta : eureka.lib.readECF.MetaClass
+        The metadata object.
+
+    Returns
+    -------
+    sy : float
+        Gaussian width in y direction.
+    sx : float
+        Gaussian width in x direction.
+    x : float
+        Refined centroid x position.
+    y : float
+        Refined centroid y position.
+
+    Notes
+    -----
+    History:
+
+    - Feb 22, 2023 Isaac Edelman 
+        Initial implementation.
+    """
+    # Create centroid position x,y 
+    # based off of centroid method 
+    # and inital centroid guess
+    if meta.centroid_tech.lower() in ['com', '1dg', '2dg']:
+        cent_func = getattr(sys.modules[__name__], 
+                            ("centroid_" + meta.centroid_tech.lower()))
+        x, y = centroid_sources(img, yxguess[1], yxguess[0], 
+                                centroid_func=cent_func, 
+                                box_size=meta.ctr_cutout_size)
+        x, y = x[0], y[0]
+    else:
+        print("Invalid centroid_tech option")
+    
+    # Cropping frame to speed up guassian fit
+    minx = -int(meta.gauss_frame)+int(x)
+    maxx = int(meta.gauss_frame)+int(x)
+
+    # Set Frame size based off of frame crop
+    frame = img[:, minx:maxx]
+
+    # Create meshgrid
+    x_shape = np.arange(img.shape[1])[minx:maxx]
+    y_shape = np.arange(img.shape[0])
+    x_mesh, y_mesh = np.meshgrid(x_shape, y_shape)
+
+    # The inital guess for miri is [400, 41, 41] 
+    # or [brightness of pixles in area of interest, xsigma, ysigma].
+    # This guess is used as a starting point for the algorithm.
+    if (meta.inst == 'nircam'):
+        initial_guess = [400, 5, 5]
+    elif (meta.inst == 'miri'):
+        initial_guess = [400, 41, 41]
```

Comment:
You have these backwards - MIRI's PSF width is ~5 pixels and NIRCam's is ~41.  Also only add parentheses to if statements if you need multiple lines of clauses. And adjust the comment to be more generic along the lines of `# The initial guess for [Gaussian amplitude, xsigma, ysigma]`